### DISCUSSION TITLE
[Bug]:

### DISCUSSION THREAD

**Discussion by afaqcentosquare** [beyondcode/herd-community#1047] [General]
### Platform

Windows

### Operating system version

Windows 11

### System architecture

Windows

### Herd Version

1.11.0

### PHP Version

php 7.4

### Bug description

I'm using Laravel Herd for a non-Laravel project, but I'm encountering an issue with custom configurations in the .conf file for my site. After making changes to the `site.test.conf` file and restarting the services, the changes don't seem to take effect. However, when I apply the same changes directly in an Nginx setup, they work fine.

Herd is installed on my `C drive`, and the configuration file I modified is located at:
`C:\Users\user\.config\herd\config\valet\Nginx\site.test.conf`

```
server {
    listen 127.0.0.1:80;
    server_name site.test www.site.test *.site.test;
    #listen VALET_LOOPBACK:80; # valet loopback
    root /;
    charset utf-8;
    client_max_body_size 128M;

    location ~* /41c270e4-5535-4daa-b23e-c269744c2f45/([A-Z]+:)(.*) {
        internal;
        alias $1;
        try_files $2 $2/;
    }

    location / {
        rewrite ^ "C:/Program Files/Herd/resources/app.asar.unpacked/resources/valet/server.php" last;
    }

    location /api/rest {
	try_files $uri $uri/ /api/rest/index.php?$query_string;
    }

    access_log off;
    error_log "C:/Users/user/.config/herd/Log/nginx-error.log";

    error_page 404 "C:/Program Files/Herd/resources/app.asar.unpacked/resources/valet/server.php";

    location ~ [^/]\.php(/|$) {
        fastcgi_split_path_info ^(.+\.php)(/.+)$;
        fastcgi_pass "$herd_sock_74";
        fastcgi_index "C:/Program Files/Herd/resources/app.asar.unpacked/resources/valet/server.php";
        include fastcgi_params;
        fastcgi_param SCRIPT_FILENAME "C:/Program Files/Herd/resources/app.asar.unpacked/resources/valet/server.php";
        fastcgi_param HERD_HOME "C:/Users/user/.config/herd";
        fastcgi_param PATH_INFO $fastcgi_path_info;
    }

    location ~ /\.ht {
        deny all;
    }
}

```

This is my additional block which I want to add.

```
    location /api/rest {
	try_files $uri $uri/ /api/rest/index.php?$query_string;
    }
    
```

But it's not working

**Comment by mpociot** [comment]
Hm this must be a configuration issue then.
If your site is isolated, and actually using this config file, nginx should pick up any custom configuration changes. Herd doesn't do anything special in this case.
